addon/services/unified-event-handler.js

```diff
@@ -218,6 +224,12 @@ export default Ember.Service.extend(Ember.Evented, {
    * @return {Void}
    */
   triggerEvent(eventName) {
-    Ember.run.throttle(this, () => this.trigger(eventName), EVENT_INTERVAL);
+    const throttleId = Ember.run.throttle(this, () => {
+      this.trigger(eventName), EVENT_INTERVAL);
+      let index = this._throttledEventTimers.indexOf(throttleId);
```

Comment:
We might want to consider using an object here to store the throttle IDs. That would make the removal O(1).

Reply:
The throttleId is an opaque token, it happens to be numeric at the moment but we will likely make it be an object in the future which would cause issues (unless we use `guidFor` or something)...

Reply:
> unless we use `guidFor` or something
Or a Map 🙂

Reply:
Indeed! Except we still support platforms without native Map...